src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -796,10 +796,10 @@ public Task State()
             }
         }
 
-        [RavenAction("/admin/connection-strings/add", "PUT", "/admin/connection-strings/add?name={databaseName:string}&type={[sql|raven]:string}")]
-        public async Task AddConnectionString()
+        [RavenAction("/admin/connection-strings/put", "PUT", "/admin/connection-strings/put?name={databaseName:string}&type={[sql|raven]:string}")]
```

Comment:
Just `/admin/connection-strings` would do, you already have `PUT` in the method.